src/components/utils/include/utils/convert_utils.h

```diff
@@ -61,6 +61,11 @@ unsigned long long int ConvertUInt64ToLongLongUInt(const uint64_t value);
 uint64_t ConvertLongLongUIntToUInt64(const unsigned long long int value);
 
 /**
+ * Return integer division round up
```

Comment:
@OHerasym 
Please, provide more detail information about parameters.
Maybe one example of usage?